pkg/apply/event/event.go

```diff
@@ -81,6 +81,7 @@ type ApplyEventType int
 const (
 	ApplyEventResourceUpdate ApplyEventType = iota
 	ApplyEventCompleted
+	ApplyEventFailed
```

Comment:
So `ApplyEventFailed` is for a single resource. For any single resource, we expect to generate either a `ApplyEventResourceUpdate` event or a `ApplyEventFailed` event?

Reply:
I realized the event hierarchy probably require some more thought, but we should try to address the asymmetry between the Prune|Delete events and the Apply event. 
I removed this from this PR, so this just addresses the printing problem. We have an existing issue to address the event hierarchy: https://github.com/GoogleContainerTools/kpt/issues/1470

Reply:
Yes, there is some inconsistency for Apply|Prune|Delete events. Let's address that in a different PR.